src/server/game/Movement/MovementGenerators/PointMovementGenerator.cpp

```diff
@@ -75,13 +75,13 @@ bool PointMovementGenerator<T>::DoUpdate(T* owner, uint32 /*diff*/)
         return true;
     }
 
-    owner->AddUnitState(UNIT_STATE_ROAMING_MOVE);
-
     if ((_interrupt && owner->movespline->Finalized()) || (_recalculateSpeed && !owner->movespline->Finalized()))
     {
         _recalculateSpeed = false;
         _interrupt = false;
 
+        owner->AddUnitState(UNIT_STATE_ROAMING_MOVE);
```

Comment:
@chaodhib : do you mean that this change belongs to a different PR, or that this line (and maybe lines below) belong in a different file (or part of this file) where a different commit was placed?